### DISCUSSION TITLE
用了好久一直没问题，今天突然提示超出额度了，请问想要继续使用要怎么办呢

### DISCUSSION THREAD

**Discussion by marypan0611** [nextai-translator/nextai-translator#790] [Q&A]
如题，提示如下：

You exceeded your current quota, please check your plan and billing details.

![スクリーンショット 2023-06-01 114643](https://github.com/openai-translator/openai-translator/assets/135190696/affcc773-a806-4faa-897f-ee5b3f1aca63)

我检查了chatgpt还是可以使用，而且18美金的额度还远远没用完，请问是怎么回事呢
谢谢

**Comment by ghost** [comment]
6月1号免费api过期了，我目前也在找方法

**Reply by ThinkerOfJS** [reply]
老哥有找到解决方法吗

**Reply by 3322youyong** [reply]
同问

**Reply by DCforwarding** [reply]
用ChatGPT登录，不要用OpenAi登录

**Comment by xmbai9507** [comment]
同问

**Comment by lamachuang** [comment]
我也遇到这个问题了，是什么意思？OPENAI的API用不了的原因？

**Comment by Sorosinthefield** [comment]
同问

**Comment by SainSmart99** [comment]
同问

**Comment by homie11** [comment]
同问

**Comment by nwuzmedoutlook** [comment]
OpenAI API和ChatGPT虽然都是由OpenAI提供的，但它们是不同的服务，使用的是不同的配额和限制。

OpenAI API是一个付费的开放接口，需要使用有效的API密钥来调用。API使用有一定的限制和配额，如果超出了你的API使用配额，你将无法继续调用API。

而ChatGPT是一个由OpenAI提供的公共演示，可以在OpenAI的网站上免费使用。OpenAI为公共演示提供了一定的配额和限制，但通常不会像API那样限制调用次数。所以在这里使用ChatGPT不会受到API调用配额的限制。

如果你在第三方客户端使用了OpenAI API，并超出了API的调用配额，但在OpenAI的网站上使用ChatGPT仍然可行，那很可能是因为你在网站上使用的是公共演示版本，而非OpenAI API。你可以确认你在第三方客户端中使用的是API密钥，并在OpenAI API的配额和限制范围内调用。

**Reply by fool-cloud** [reply]
那如果还想要继续使用请问应该怎么做？

**Reply by Cvxiaozhu** [reply]
充钱！！

**Comment by chengfengfengwang** [comment]
> 那如果还想要继续使用请问应该怎么做？
去open AI官网充值

**Comment by capalens** [comment]
我有api，每个月5美元额度，但是我使用的时候，翻译告诉无法access，请问是什么问题？

**Reply by capalens** [reply]
刷了一下以后，连submit的键也消失了。

**Comment by JingQingDaYe** [comment]
24年6月份,还是不能用：
You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.

**Reply by Lucymoguoquan** [reply]
绑定支付方式，然后开启付费计划就可以了。要信用卡，国内可以申请虚拟信用卡：https://1VHJTG.fomepay.org

**Reply by Wil-79** [reply]
> 绑定支付方式，然后开启付费计划就可以了。要信用卡，国内可以申请虚拟信用卡：https://1VHJTG.fomepay.org

好像注册不了了，说联系管理员
